tests/test_decorators.py

```diff
@@ -73,49 +116,57 @@ def test_success_is_failure_tree():
     assert(root.status == py_trees.common.Status.FAILURE)
 
 
-def test_inverter_tree():
-    console.banner("Inverter Tree")
+def test_inverter():
+    console.banner("Inverter")
     root = py_trees.composites.Sequence(name="Root")
     selector = py_trees.composites.Selector(name="Selector")
     failure = py_trees.behaviours.Failure(name="Failure")
-    failure2 = py_trees.meta.inverter(py_trees.behaviours.Success)(name="Failure2")
+    failure2 = py_trees.decorators.Inverter(child=py_trees.behaviours.Success())
```

Comment:
how about renaming it to success_inverter

Reply:
Done